### ISSUE TITLE
There is no code folder in master

### ISSUE THREAD

**Issue by aleksey-sinos** [rlabbe/Kalman-and-Bayesian-Filters-in-Python#152]
Hello, Rlabbe!
Thank you for the exellent book and library!
Unfortunally I was unable to run code directly from master, becouse there is no code folder in branch.
Is it a mistake or I've done somethink wrong?

**Comment by senden9** [comment]
The folder `code` is renamed to `kf_book`. See issue #138 that is fixed by commit 3d288d60d617616e44257dec25ed1a8ca4ad2bcf if you want to know the reason.

**Comment by rlabbe** [comment]
What lead you to look for the /code subdirectory? Are their instructions in the book saying to use the code subdirectory, or is some code still referring to that subdirectory?

**Comment by aleksey-sinos** [comment]
Your book was my first look on Jupyter Notebook, so it is probably my misunderstanding. 
First I was using binder to study your book (with code folder). Then I tried to download notebook from binder and run it on a local mashine. The run was unsuccsesfull and I thought that is because there is no code folder in the local folder of the book.
Actually I still don not know how to run modified binder notebook locally, but it isn't so important, becouse I've already made the same experiments at the local notebook.
